java/src/jmri/jmrit/operations/trains/TrainBuilderCars.java

```diff
@@ -1230,6 +1232,8 @@ private boolean sendCarToDestinationSpur(Car car, Track track) {
      */
     private boolean sendCarToDestinationTrack(Car car, Track track) {
         if (!checkBasicMoves(car, track)) {
+            addLine(_buildReport, SEVEN, MessageFormat.format(Bundle.getMessage("trainCanNotDeliverToDestination"),
```

Comment:
No need to change this, but `Bundle.getMessage(message, ....)` can take multiple arguments and do the `Message.format(..)` operation for you.

Reply:
Didn't know that. Thanks